manim/mobject/coordinate_systems.py

```diff
@@ -59,6 +59,44 @@
 class CoordinateSystem:
     """
     Abstract class for Axes and NumberPlane
+
+    Examples
+    --------
+
+    .. manim:: CoordSysExample
+        :save_last_frame:
+
+        class CoordSysExample(Scene):
+            def construct(self):
+                # When drawing Axes, the location of the ticks
+                # on the axes depend on x_range and y_range.
+                grid = Axes(
+                    x_range=[0, 1, 0.05],
+                    y_range=[0, 1, 0.05],
+                    x_length=9,
+                    y_length=5.5,
+                    axis_config={"decimal_number_config": {"num_decimal_places": 2}},
```

Comment:
Since #1667 was merged to automatically determine `num_decimal_places`, this argument actually isn't necessary. There's some value for including defaults, but I think it's better to show off this feature than have the user think that this must always be included. 

Maybe including a comment that step size determines `num_decimal_places`?

Reply:
My latest commit should have fixed this, however, I was not able to verify if the numbers on the x- and y- axis were actually correct because I am still running v0.7.0.